ui-731-ticket-ny-onja

ny-sowell opened the pull request.

nfanorenana committed:
ui-731-ticket-ny-onja

nfanorenana committed:
ui-735-creer-un-component-pour-afficher-les-infos-de-chaque-user

fandresena-sowell reviewed (changes requested):
- Utiliser computed pour déterminer la liste à afficher
- Mieux nommer les variables
- Utiliser v-for="(item, index)

nfanorenana committed:
ui-735-creer-un-component-pour-afficher-les-infos-de-chaque-user

ui-735-creer-un-component-pour-afficher-les-infos-de-chaque-user

- Changer UserCard en ListItem
- Modifier les props de ListItem

nfanorenana committed:
ui-731-ticket-ny-onja

- Modification des noms de variable
- Utilisation de computed
- Utilisation de v-for="(item, index)...

fandresena-sowell committed:
Merge pull request #16 from SoWellApp/ui-735-creer-un-component-pour-afficher-les-infos-de-chaque-user

ui-735-creer-un-component-pour-afficher-les-infos-de-chaque-user
